include/dpp/channel.h

```diff
@@ -158,6 +187,51 @@ struct DPP_EXPORT thread_member
 	 thread_member& fill_from_json(nlohmann::json* j);
 };
 
+/**
+ * @brief Specifies the emoji to use as the default way to react to a forum post. Exactly one of `emoji_id` and `emoji_name` must be set.
+ */
+struct DPP_EXPORT default_reaction_emoji {
```

Comment:
does this need a ctor?

Reply:
no it doesnt, its a simple struct with types that can be easily initialised